tests/testCampaign.py

```diff
@@ -0,0 +1,112 @@
+from deltamail.campaign import BulkMailCampaign, TransactionMailCampaign
```

Comment:
What about tests for the command line usage?
Don't need to add it now, add a TODO.